sdk/eventhub/event-hubs/README.md

```diff
@@ -1,259 +1,198 @@
-```
-- If you are interested only in **errors**, then you can set the `DEBUG` environment variable as follows:
-```bash
-export DEBUG=azure:event-hubs:error,azure-amqp-common:error,rhea-promise:error,rhea:events,rhea:frames,rhea:io,rhea:flow
+npm install @types/node
 ```
 
-#### Logging to a file
-- Set the `DEBUG` environment variable as shown above and then run your test script as follows:
-  - Logging statements from your test script go to `out.log` and logging statements from the sdk go to `debug.log`.
-    ```bash
-    node your-test-script.js > out.log 2>debug.log
-    ```
-  - Logging statements from your test script and the sdk go to the same file `out.log` by redirecting stderr to stdout (&1), and then redirect stdout to a file:
-    ```bash
-    node your-test-script.js >out.log 2>&1
-    ```
-  - Logging statements from your test script and the sdk go to the same file `out.log`.
-    ```bash
-      node your-test-script.js &> out.log
-    ```
+You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your tsconfig.json. Note that if you have enabled `compilerOptions.esModuleInterop`, `allowSyntheticDefaultImports` is enabled by default. See [TypeScript's compiler options handbook](https://www.typescriptlang.org/docs/handbook/compiler-options.html) for more information.
+
+### Authenticate the client
+
+Interaction with Event Hubs start with an instance of the [EventHubClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest) class. You can instantiate
+this class using one of the 4 static methods on it
 
-## Examples ##
+- [createFromConnectionString](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromconnectionstring-string--ClientOptions-)
+  - This method takes the connection string and entity name to your Event Hub instance. You can get the connection string
+    from the Azure portal
+- [createFromIotHubConnectionString](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromiothubconnectionstring-string--clientoptions-)
+  - This method takes an IotHub Connection string. You can get the connection string from the Azure portal. This is useful for receiving telemetry data
+    of IotHub from the linked EventHub. Most likely the associated connection string will not have send claims. Hence getting HubRuntimeInfo or PartitionRuntimeInfo
+    and receiving events would be the possible operations.
+- [createFromTokenProvider](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest#createfromtokenprovider-string--string--tokenprovider--clientoptionsbase-)
+  - This method takes the host name and entity name of your Event Hub instance and your custom Token Provider. The
+    host name is of the format `name-of-event-hub-instance.servicebus.windows.net`.
```

Comment:
Are we sure about "servicebus" here?

Reply:
Yes, this is the format of host name`<yournamespace>.servicebus.windows.net` in Event hubs.